dintero-checkout-for-woocommerce.php

```diff
@@ -234,12 +234,21 @@ function () {
 		 * @return void
 		 */
 		public function wc_zero_decimal_notice() {
+
+			if ( get_user_meta( get_current_user_id(), 'dismissed_wc_zero_decimal_notice', true ) ) {
+				return;
+			}
+
 			$decimals_setting = get_option( 'woocommerce_price_num_decimals', 2 );
 
 			if ( 0 == $decimals_setting ) {
 				$article_url = 'https://krokedil.com/dont-display-prices-with-0-decimals-in-woocommerce/';
 				?>
-				<div class="notice notice-warning is-dismissible">
+				<div class="updated woocommerce-message">
```

Comment:
The type of message was changed to updated? Is there a reason for that instead of having it as a notice?